Vladyslav_Gurevych/HW7/Task2.py

```diff
@@ -0,0 +1,19 @@
+from math import sqrt
+
+
+def get_square_parameters(side_of_square: float) -> tuple:
+    """
+        Takes side of the square, and returns a tuple of perimeter, area, diagonal
+    """
+    perimeter = round(side_of_square * 4, 2)
+    area = round(side_of_square ** 2)
+    diagonal = round(sqrt(2) * side_of_square, 2)
+    result_tuple = (perimeter, area, diagonal)
+    return result_tuple
```

Comment:
you can just code:
return perimeter, area, diagonal

instead of:
result_tuple = (perimeter, area, diagonal)
    return result_tuple

Reply:
done